nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ScanAttribute.java

```diff
@@ -97,13 +107,23 @@
             .addValidator(StandardValidators.createRegexValidator(0, 1, false))
             .defaultValue(null)
             .build();
+    public static final PropertyDescriptor DICTIONARY_ENTRY_METADATA_DEMARCATOR = new PropertyDescriptor.Builder()
+            .name("Dictionary Entry Metadata Demarcator")
```

Comment:
ok, cool.  i copied and pasted the existing properties which have not implemented displayName yet.  Do you want me to change it for the new properties that I added and leave the existing properties alone?